.gitpod.Dockerfile

```diff
@@ -0,0 +1,27 @@
+#Pull Gitpod Workspace Image
```

Comment:
I just noticed that but could you move the file to a `.gitpod` folder and name it only `Dockerfile`?